provision/Rakefile

```diff
@@ -67,7 +67,7 @@ class Redhat
   end
 
   def uninstall(pkg_name,pkg_verion)
-    sh("yum -e --assumeyes #{pkg_name}-#{pkg_verion}")
```

Comment:
if this is taking more than 5 seconds, `rpm -e PACKAGE_NAME` will do the trick a lot faster, no need to specify version.